packages/devtools_app/test/flutter/inspector_screen_test.dart

```diff
@@ -0,0 +1,163 @@
+    }
+
+    testWidgets('builds its tab', (WidgetTester tester) async {
+      await tester.pumpWidget(wrap(Builder(builder: screen.buildTab)));
+      expect(find.text('Flutter Inspector'), findsOneWidget);
+    });
+
+    testWidgets('builds with no data', (WidgetTester tester) async {
+      // Make sure the window is wide enough to display description text.
+      await setWindowSize(const Size(1920.0, 1200.0));
+      await tester.pumpWidget(wrap(Builder(builder: screen.build)));
+      expect(find.byType(InspectorScreenBody), findsOneWidget);
+      expect(find.text(extensions.toggleSelectWidgetMode.description),
+          findsOneWidget);
+      expect(find.text(extensions.debugPaint.description), findsOneWidget);
+      // Make sure there is not an overflow if the window is shrunk.
+      await setWindowSize(const Size(600.0, 1200.0));
+      // Verify that description text is no-longer shown.
+      expect(find.text(extensions.debugPaint.description), findsOneWidget);
+    });
+
+    testWidgets('Test toggling service extension buttons',
+        (WidgetTester tester) async {
+      await setWindowSize(const Size(1920.0, 1200.0));
+      mockExtensions();
+      expect(
+          fakeExtensionManager
+              .extensionValueOnDevice[extensions.debugPaint.extension],
+          isFalse);
+      expect(
+          fakeExtensionManager.extensionValueOnDevice[
+              extensions.toggleSelectWidgetMode.extension],
+          isTrue);
+
+      await tester.pumpWidget(wrap(Builder(builder: screen.build)));
+      expect(find.byType(InspectorScreenBody), findsOneWidget);
+      expect(find.text(extensions.toggleSelectWidgetMode.description),
+          findsOneWidget);
+      expect(find.text(extensions.debugPaint.description), findsOneWidget);
+      await tester.pumpAndSettle();
+
+      expect(
+          fakeExtensionManager.extensionValueOnDevice[
+              extensions.toggleSelectWidgetMode.extension],
+          isTrue);
+      await tester
+          .tap(find.text(extensions.toggleSelectWidgetMode.description));
+      expect(
+          fakeExtensionManager.extensionValueOnDevice[
+              extensions.toggleSelectWidgetMode.extension],
+          isFalse);
+      // Verify the the other service extension's state hasn't changed.
+      expect(
+          fakeExtensionManager
+              .extensionValueOnDevice[extensions.debugPaint.extension],
+          isFalse);
+
+      await tester
+          .tap(find.text(extensions.toggleSelectWidgetMode.description));
+      expect(
+          fakeExtensionManager.extensionValueOnDevice[
+              extensions.toggleSelectWidgetMode.extension],
+          isTrue);
+
+      await tester.tap(find.text(extensions.debugPaint.description));
+      expect(
+          fakeExtensionManager
+              .extensionValueOnDevice[extensions.debugPaint.extension],
+          isTrue);
+    });
+
+    testWidgets('Test toggling service extension buttons',
```

Comment:
this is the same test name as the previous test case. Please modify to describe difference.

Reply:
@pq would be nice to have a lint if you have two tests in the same group with the same name. :)
done.